ontology/yaml/resources/states/states.yaml

```diff
@@ -25,6 +25,8 @@ OCCUPIED: Occupied sensor state or operation mode.
 UNOCCUPIED: Unoccupied sensor state or operation mode.
 VFD: Discrete speed (frequency) control
 BYPASS: Bypassing an element or entity.
+DRY: Modes of operation in DX unit
```

Comment:
Insufficient definition. Please elaborate and clarify.

Reply:
The air-conditioner in dry mode acts as a dehumidifier by removing moisture from the indoor air.

Reply:
that needs to be added to the definition.

Reply:
and this should probably be DEHUMIDIFYING or something similar.